"""
Here is another illustrative example of a runtime error. Run the follow code
and study the error that it generates. Can you locate the problem?
"""
print( ((2*3) /4 + (5-6/7) *8 )         # lost a char ) last line
print( ((12*13) /14 + (15-16)/17) *18 )